plugin.xml

```diff
@@ -52,6 +68,8 @@
     <source-file src="src/android/AuthPendingResult.java" target-dir="src/edu/berkeley/eecs/emission/cordova/jwtauth"/>
     <source-file src="src/android/AuthTokenCreator.java" target-dir="src/edu/berkeley/eecs/emission/cordova/jwtauth"/>
     <source-file src="src/android/AuthTokenCreationFactory.java" target-dir="src/edu/berkeley/eecs/emission/cordova/jwtauth"/>
+    <source-file src="src/android/AuthStateManager.java" target-dir="src/edu/berkeley/eecs/emission/cordova/jwtauth"/>
```

Comment:
maybe rename to `OpenIDAuthStateManager` to indicate that this is OpenID specific...